spec/models/service_spec.rb

```diff
@@ -77,6 +78,36 @@
 
         it { is_expected.to be true }
       end
+
+      context 'when service is probation service' do
+        let(:service) { build(:service, ending: ending, service_type: :probation) }
+        let(:ending) { Time.zone.today.at_end_of_week - 2.days }
+
+        context 'when ending is a friday' do
+          it { is_expected.to be false }
+        end
```

Comment:
Since you have declared an extra context for that, I'd move the let
```suggestion
        context 'when ending is a friday' do
          let(:ending) { Time.zone.today.at_end_of_week - 2.days }

          it { is_expected.to be false }
        end
```